src/main/java/stsjorbsmod/relics/BottledBurdenRelic.java

```diff
@@ -0,0 +1,153 @@
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.PowerTip;
+import com.megacrit.cardcrawl.localization.LocalizedStrings;
+import com.megacrit.cardcrawl.rooms.AbstractRoom;
+import stsjorbsmod.actions.ExhumeCardsAction;
+import stsjorbsmod.patches.EntombedField;
+import stsjorbsmod.patches.SelfExertField;
+import stsjorbsmod.patches.SelfExhumeFields;
+import stsjorbsmod.util.CardUtils;
+
+import java.util.Iterator;
+import java.util.function.Predicate;
+
+import static stsjorbsmod.JorbsMod.makeID;
+import static stsjorbsmod.characters.Cull.Enums.CULL_CARD_COLOR;
+import static stsjorbsmod.patches.BottledBurdenPatch.AbstractCardMemoryFields.inBottledBurden;
+
+public class BottledBurdenRelic extends CustomJorbsModRelic implements CustomBottleRelic, CustomSavable<Integer> {
+    public static final String ID = makeID(BottledBurdenRelic.class);
+    public AbstractCard card = null;
+    public static final int EXHUME_TURN = 3;
+    private boolean cardSelected = true;
+
+    public BottledBurdenRelic(){super(ID, CULL_CARD_COLOR, RelicTier.COMMON, LandingSound.CLINK);}
+
+    @Override
+    public Predicate<AbstractCard> isOnCard() {
+        return inBottledBurden::get;
+    }
+
+    @Override
+    public Integer onSave() {
+        if (card != null) {
+            return AbstractDungeon.player.masterDeck.group.indexOf(card);
+        } else {
+            return -1;
+        }
+    }
+
+    @Override
+    public void onLoad(Integer cardIndex) {
+        if (cardIndex == null) {
+            return;
+        }
+        if (cardIndex >= 0 && cardIndex < AbstractDungeon.player.masterDeck.group.size()) {
+            card = AbstractDungeon.player.masterDeck.group.get(cardIndex);
+            if (card != null) {
+                inBottledBurden.set(card, true);
+                AbstractCard cardInDeck = AbstractDungeon.player.masterDeck.getSpecificCard(this.card);
+                EntombedField.entombed.set(cardInDeck, true);
+                setDescriptionAfterLoading();
+            }
+        }
+    }
+
+    @Override
+    public void onEquip() {
+        // follow same behavior as base game bottle relics.
+        CardGroup nonExertCards = CardUtils.getCardsForBottling(AbstractDungeon.player.masterDeck.getPurgeableCards());
+        nonExertCards.group.removeIf(c -> SelfExertField.selfExert.get(c));
+        if (nonExertCards.size() > 0) {
+            this.cardSelected = false;
+            if (AbstractDungeon.isScreenUp) {
+                AbstractDungeon.dynamicBanner.hide();
+                AbstractDungeon.overlayMenu.cancelButton.hide();
+                AbstractDungeon.previousScreen = AbstractDungeon.screen;
+            }
+
+            AbstractDungeon.getCurrRoom().phase = AbstractRoom.RoomPhase.INCOMPLETE;
+            AbstractDungeon.gridSelectScreen.open(nonExertCards, 1, DESCRIPTIONS[1] + this.name + LocalizedStrings.PERIOD, false, false, false, false);
```

Comment:
For the string from DESCRIPTIONS, I wonder if it's better to let this be formatted instead of concatenated.

Reply:
I followed the pattern for Bottled Memory, but I'm down to change both. I didn't want to mess with that existing pattern if there was a specific reason we did it that way. Can't think of anything.